src/reports/components/ReportsLayout.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Outlet, useLocation } from 'react-router-dom';
+import PageLayout from '../../common/components/PageLayout';
+import ReportsMenu from './ReportsMenu';
+
+const reportsPaths = {
+  '/reports/combined': ['reportTitle', 'reportCombined'],
+  '/reports/route': ['reportTitle', 'reportRoute'],
+  '/reports/event': ['reportTitle', 'reportEvents'],
+  '/reports/trip': ['reportTitle', 'reportTrips'],
+  '/reports/stop': ['reportTitle', 'reportStops'],
+  '/reports/summary': ['reportTitle', 'reportSummary'],
+  '/reports/chart': ['reportTitle', 'reportChart'],
+  '/reports/logs': ['reportTitle', 'statisticsTitle'],
+  '/reports/scheduled': ['settingsTitle', 'reportScheduled'],
+  '/reports/statistics': ['reportTitle', 'statisticsTitle'],
```

Comment:
You probably don't need to duplicate `'reportTitle'` everywhere. Just the second part is different.

Reply:
I made this change, what do you think? 395ada36c39d5a003b6e0b81242039cd4e62d571

Reply:
I found that being part was different->  '/reports/scheduled': ['settingsTitle', 'reportScheduled'],

Reply:
This is probably a mistake. All of them should be under reports.